app/models/user.rb

```diff
@@ -14,7 +14,6 @@
 #  app_uuid               :string
 #  last_post              :datetime
 #  role                   :integer          default("0")
-#  store_owner_code       :string
```

Comment:
where did the store_owner_code go?

Reply:
your db version might not be up to date, I brought back the `store_owner_code`

Reply:
:facepalm: `annotate` requires me to run `db:migrate`. it does not read from schema.rb. I'll make a new PR adding it back.

Reply:
I added it =) tis cool